frontend/js/src/settings/import/ImportListens.tsx

```diff
@@ -69,13 +69,23 @@ function renderImport(
           <dd className="col-8">{im.import_id}</dd>
           <dt className="col-4">File name</dt>
           <dd className="col-8">{im.metadata.filename}</dd>
+          <dt className="col-4">Service</dt>
+          <dd className="col-8">
+            {Services[(im.service as unknown) as keyof typeof Services]}
+          </dd>
           <dt className="col-4">Start date</dt>
-          <dd className="col-8">{format(im.from_date, "PPP")}</dd>
+          <dd className="col-8">
+            {im.from_date && new Date(im.from_date).getTime() !== 0
```

Comment:
You could use the `isValid` utility function from date-fns, since we're already using the `format` function:

```suggestion
            {isValid(im.from_date)
```

Same for the other date below